Integrations/integration-C2sec.yml

```diff
@@ -0,0 +1,246 @@
+    if 'domain' in demisto.args():
+      DOMAIN = demisto.args()['domain']
+    else:
+        DOMAIN = demisto.params()['domainName']
+    if demisto.params()['endpointURL'].count("/") == 3:
+      ENDPOINTURL = demisto.params()['endpointURL']+ "/"
+    else:
+        ENDPOINTURL = demisto.params()['endpointURL']
+    USE_SSL = not demisto.params().get('unsecure', False)
+
+    def addcompany(domain, newscan):
+       if newscan == 'true':
+         newscan = 1
+       else:
+           newscan = 0
+       params = {'apikey': API_KEY, 'domain': domain, 'newscan': newscan}
+       call = requests.get(ENDPOINTURL + 'iriskaddcompany', headers=HEADERS, verify=USE_SSL, params=params)
+       if call.status_code == requests.codes.ok:
+          md = tableToMarkdown('Add Company Result', call.json())
+          return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.AddCompany": call.json() }
+                }
+       else:
+            return ('Error adding a new company - status code: [%d] - reason: %s' % (call.status_code, call.text))
+
+
+
+    def queryWorkitem(workitemid):
+       params = {'apikey': API_KEY, 'workitemid': workitemid}
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(ENDPOINTURL + '/iriskqueryapistatus', headers=HEADERS, verify=USE_SSL, params=params)
+       if call.status_code == requests.codes.ok:
+         resp = call.json()
+         resp['apistatus']['workitemid'] = workitemid
+         md = tableToMarkdown('Query WorkItem Result', resp['apistatus'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': resp,
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.Scan(val.workitemid && val.workitemid == obj.workitemid)": resp['apistatus']}
+                }
+       else:
+           return ('Error querying a workitem [%d] - reason: %s' % (call.status_code, call.text))
+
+    def queryIssues(domain):
+       params = {'apikey': API_KEY, 'domain': domain }
+       headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+       call = requests.get(ENDPOINTURL + 'iRiskQueryIssues', headers=headers, verify=USE_SSL, params=params)
+       if call.status_code == requests.codes.ok:
+         md = tableToMarkdown('Query Issues Result', call.json()['issueList'])
+         return  {
+                    'Type': entryTypes['note'],
+                    'ContentsFormat': formats['json'],
+                    'Contents': call.json(),
+                    'HumanReadable': md,
+                    'EntryContext': {"C2sec.Issues": call.json()['issueList'] }
```

Comment:
```suggestion
                    'EntryContext': {"C2sec.Issue": call.json()['issueList'] }
```

Reply:
@ronykoz  - I use one context key for all the issues I may pull out from c2sec. let's leave it like this.